client/src/components/FollowList/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react';
+import { Modal, ModalHeader, ModalBody, Button } from 'reactstrap';
+import userphoto from '../../assets/user1.png';
+import './style.scss';
+interface IFollowData {
+    username: string;
+}
+export const FollowList = ({toggle, modal, title,data}: {toggle: () => void,
+    modal: boolean, title: string, data: IFollowData[]}): ReactElement=> (
+            <div>
+                <Modal className='modal-dialog-centered followers-modal' isOpen={modal}>
+                    <ModalHeader className='modal-header' toggle={toggle}
+                     cssModule={{'modal-title': 'w-100 text-center '}}>
```

Comment:
Remove casModule pls

Reply:
I cannot center text in built in modalheader otherwise.